helloworld/package.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "helloworld",
+  "version": "1.0.0",
+  "description": "node code for lab",
+  "main": "hello.js",
+  "scripts": {
+    "lint": "./node_modules/eslint/bin/eslint.js .",
+    "test": "./node_modules/mocha/bin/mocha.js ."
```

Comment:
The test script is still not quite right. Here's a copy pasted version of the scripts:

```"scripts": {
    "lint": "./node_modules/eslint/bin/eslint.js .",
    "test": "./node_modules/mocha/bin/mocha"
  },```